packages/good-design/src/advanced/web3action/Web3Action.tsx

```diff
@@ -71,7 +71,7 @@ export const Web3ActionButton = ({
       if (handleConnect) {
         await handleConnect();
       } else {
-        activateBrowserWallet();
+        await activateBrowserWallet();
```

Comment:
Why ? It is not async - we have an issue because of await

Reply:
https://github.com/TrueFiEng/useDApp/blob/389756bcbb7646e3bb17716f1c236e150d551177/packages/core/src/providers/network/connectors/context.tsx#L78

I guess the type annotation is wrong, because it is actually async.